issuer-service/src/main/java/ch/admin/bj/swiyu/issuer/service/MetadataService.java

```diff
@@ -7,28 +7,36 @@
 import ch.admin.bj.swiyu.issuer.domain.credentialoffer.ConfigurationOverride;
 import ch.admin.bj.swiyu.issuer.domain.openid.metadata.IssuerMetadata;
 import ch.admin.bj.swiyu.issuer.service.factory.strategy.KeyStrategyException;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
 import com.nimbusds.jose.*;
 import com.nimbusds.jwt.JWTClaimsSet;
 import com.nimbusds.jwt.SignedJWT;
 import lombok.AllArgsConstructor;
+import lombok.RequiredArgsConstructor;
 import lombok.extern.slf4j.Slf4j;
 import org.apache.commons.lang3.time.DateUtils;
+import org.springframework.aop.framework.Advised;
+import org.springframework.aop.framework.AopProxyUtils;
 import org.springframework.stereotype.Service;
 
+import java.text.ParseException;
 import java.util.Date;
-import java.util.Map;
 import java.util.UUID;
 
 @Service
-@AllArgsConstructor
+@RequiredArgsConstructor
 @Slf4j
 public class MetadataService {
 
     private final OpenIdIssuerConfiguration openIdIssuerConfiguration;
     private final CredentialManagementService credentialManagementService;
     private final SignatureService signatureService;
+    private final EncryptionService encryptionService;
+    private final DemonstratingProofOfPossessionService demonstratingProofOfPossessionService;
     private final SdjwtProperties sdjwtProperties;
     private final ApplicationProperties applicationProperties;
+    private final ObjectMapper objectMapper = new ObjectMapper();
```

Comment:
Do you need to instantiate this? Shouldn't:
```suggestion
    private final ObjectMapper objectMapper;
```

be enough?